nucleus/grizzly/config/src/main/java/org/glassfish/grizzly/config/dom/Http.java

```diff
@@ -112,7 +112,7 @@ public interface Http extends ConfigBeanProxy, PropertyBag {
     float HTTP2_CLEAN_PERCENTAGE = 0.5f;
     int HTTP2_CLEAN_FREQUENCY_CHECK = 50;
     boolean HTTP2_DISABLE_CIPHER_CHECK = false;
-    boolean HTTP2_PUSH_ENABLED = false;
+    boolean HTTP2_PUSH_ENABLED = true;
```

Comment:
If I remember right we ship with http2 push disabled on purpose; server should be configured with push enabled for tck

Reply:
Added configuration setting to TCK Runer:
````
<exec executable="${glassfish.asadmin}" dir="${glassfish.home}/glassfish/bin">
  <arg value="set" />
  <arg value="configs.config.server-config.network-config.protocols.protocol.http-listener-1.http.http2-push-enabled=true" />
</exec>
````